languages/csharp-dotnet-core2/template/Algorithm.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please ensure that your algorithm only targets a single framework, multiple frameworks are not supported. -->

```

Comment:
This is due to how our config.json file works, I can't imagine a world where this matters, but something to keep in mind.